tools/sds-aspect-model-maven-plugin/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
+  ~
+  ~ See the AUTHORS file(s) distributed with this work for additional
+  ~ information regarding authorship.
+  ~
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+  ~
+  ~ SPDX-License-Identifier: MPL-2.0
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+   <parent>
+      <artifactId>sds-sdk-parent</artifactId>
+      <groupId>io.openmanufacturing</groupId>
+      <version>DEV-SNAPSHOT</version>
+      <relativePath>../../pom.xml</relativePath>
+   </parent>
+   <modelVersion>4.0.0</modelVersion>
+
+   <artifactId>sds-aspect-model-maven-plugin</artifactId>
+   <packaging>maven-plugin</packaging>
+   <name>SDS Aspect Model Maven Plugin</name>
+
+   <properties>
+      <maven-plugin-annotations-version>3.5</maven-plugin-annotations-version>
```

Comment:
Why do you not declare it in the parent pom?

Reply:
I added these properties here, since the corresponding dependencies will never be used in any other module.

Reply:
Yes maybe, but it's harder to update, because nobody looks in the sub pom to upgrade deps. This is mey experience :)